CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +16,27 @@ internal class Testing
          */
 
         //Method
+        public static void DieTest()//TestDie method
+        {
+            Die die = new Die();
+            int outcome = die.Roll(); //creates a die to be tested
+
+            Debug.Assert(outcome < 1 || outcome >= 7, "Die test failed");// compares the output of the DieTEst
+            
```

Comment:
The < > operators are the wrong way around as Debug.Assert() will display "Die test failed" when the outcome is between 1 and 7